website/docs/r/instance_server.html.markdown

```diff
@@ -57,7 +57,7 @@ resource "scaleway_instance_server" "web" {
   tags = [ "hello", "public" ]
 
   ip_id = scaleway_instance_ip.ip.id
-  disable_dynamic_ip = true # required for now when using `ip_id`
```

Comment:
The whole line can be removed as `enable_dynamic_ip = false` is the default value for this attribute

Reply:
Oh yes of course :)